dockerunit-consul/src/main/java/com/github/qzagarese/dockerunit/discovery/consul/ConsulDescriptor.java

```diff
@@ -23,12 +25,13 @@
 	static final int CONSUL_DNS_PORT = 8600;
 	static final int CONSUL_PORT = 8500;
 
-	@ContainerBuilder
+    private static final Logger logger = Logger.getLogger(ConsulDescriptor.class.getSimpleName());
+
+
+    @ContainerBuilder
```

Comment:
This class seems a mix of tab and spaces indentation, it should use 1 and stick with it (by 1 I mean spaces, tab indentation is the work of the devil :smiling_imp:)

Reply:
You are right :-D Would the default IntelliJ work?